sections/security/dependencysecurity.md

```diff
@@ -0,0 +1,50 @@
+
+### Table of Contents
+- [Node Security Platform (nsp)](#node-security-platform)
+- [Snyk](#snyk)
+- [Greenkeeper](#greenkeeper)
+
+### Node Security Platform (nsp)
+Node Security Platform is a set of services targeted towards dependency management with a focus on security.
+
+The most common use of this tool is the service nsp Live. This is a service which integrates into a project's CI workflow and Github Pull Requests to identify vulnerabilities present in the project. The benefits of this mean that when new modules are added, any vulnerabilites are automatically identified in order for the security risk to be mitigated before these dependencies reach master. This also includes new vulnerabilites which are found in existing dependencies, as well as when new dependencies are introduced.
+
+There is also an nsp CLI tool, which traverses your dependencies checking for vulnerabilities against the advisories list. To use this tool:
+Install the module globally.
+`npm install -g nsp`
+Run the CLI tool from the root of the project directory.
+`nsp check`
+An example of output of nsp check
+![nsp check example](https://i.stack.imgur.com/uvxjQ.png)
+
+🔗 [Read on: NSP website](https://nodesecurity.io/)
+
+### Snyk
+Snyk offers a feature-rich CLI, as well as Github integration. Snyk goes further with this and in addition to notifying vulnerabilities, also automatically creates new pull requests fixing vulnerabilities as patches are released for known vulnerabilties.
+
+Snyk's feature rich website also allows for ad-hoc assessment of dependencies when provided with a GitHub repository or npm module url. You can also search for npm packages which have vulnerabilties directly https://snyk.io/vuln?packageManager=npm
+An example of the output of this tool [can be found here](https://snyk.io/test/npm/passport), which shows the current vulnerability status of the latest Passport.js npm package. 
+
+🔗 [Read on: Snyk website](https://snyk.io/)
+
+### Greenkeeper
+Greenkeeper is a service which offers real-time dependency updates, which keeps an application more secure by always using the most update to date and patched dependency versions.
+
+Greenkeeper watches the npm dependencies specified in a repository's `package.json` file, and automatically creates a working branch with each dependency update. The repository CI suite is then run to reveal any breaking changes for the updated dependency version in the application. If CI fails due the dependency update, a clear and consise issue is created in the repository to be actioned outlining the current and updated package versions, along with information and commit history of the updated version.
+
+🔗 [Read on: Greenkeeper website](https://greenkeeper.io/)
+
+### Additional resources
+
```

Comment:
I would add one image of nsp/snyk results - a picture worth thousand words... :)